bench-tps/src/bench.rs

```diff
@@ -306,7 +309,11 @@ fn generate_system_txs(
         .par_iter()
         .map(|(from, to)| {
             (
-                system_transaction::transfer(from, &to.pubkey(), 1, *blockhash),
+                if !dummy_sig {
```

Comment:
- [ ] TODO: I'm pretty sure there is still a few places to be branched like this in the `solana-bench-tps`. Once sanctioned, I'm work on it to finish this.